packages/react/react/src/utils/convertNewlineToJSX.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { convertNewlineToJSX } from './convertNewlineToJSX';
+
+describe('convertNewlineToJSX', () => {
+  it('converts newline characters into <br /> JSX elements', () => {
```

Comment:
When creating a test case

I think it would be good to write 0, 1, n + 1 case.

Currently, it seems that only the situation with 1` \n` has been verified.
Can you write test cases when there is 0 `\n` test case at all or more than one `\n`?

```tsx

// zero \n
const input = 'There was a problem trying to sign in. please try again.';

// n + 1 \n
const input = 'There was a problem \ntrying to\n sign in. please\ntry again.';

```